src/ert/storage/local_storage.py

```diff
@@ -272,6 +272,10 @@ def close(self) -> None:
         the storage.
         """
 
+        for ens in self._ensembles.values():
+            ens.unify_responses()
```

Comment:
Guess this is safe to do?

Reply:
Yep, new invocations will have no effect if there are no "new" datasets in realization folders. If there are some, they will overwrite (for those given realizations) into the combined, before being deleted.